<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DXF FEM Calculator 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>455</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter 'help' for assistance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearScreenButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>176</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Screen</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>683</width>
      <height>247</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="6">
      <widget class="QPushButton" name="showMeshButton">
       <property name="text">
        <string>Mesh is OFF</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="astartendlabel">
       <property name="text">
        <string>start, end (m)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="showNodalValuesButton">
       <property name="text">
        <string>Nodal Values is OFF</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="elSizeSlider">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>250</number>
       </property>
       <property name="sliderPosition">
        <number>250</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="showGeometryButton">
       <property name="text">
        <string>Geometry is OFF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSpinBox" name="paramStep"/>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="paramButton">
       <property name="text">
        <string>Param Study is OFF</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QRadioButton" name="paramVaryBRadio">
       <property name="text">
        <string>Vary</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="unitsLabel">
       <property name="text">
        <string>Units</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="bstartendlabel">
       <property name="text">
        <string>start, end (m)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="showElementValuesButton">
       <property name="text">
        <string>Element Values is OFF</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Param Steps</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="elSizeLabel">
       <property name="text">
        <string>Max El Size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="bEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="bLabel">
       <property name="text">
        <string>Param B Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="aLabel">
       <property name="text">
        <string>Param A Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tEdit"/>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="aEndEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="aNameEdit"/>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="bEndEdit"/>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="aEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="bNameEdit"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="d3Edit"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="d2Edit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="d1Edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="d3Label">
       <property name="text">
        <string>Dim 3 Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="d2Label">
       <property name="text">
        <string>Dim 2 Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="d1Label">
       <property name="text">
        <string>Dim 1 Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="d1ValueLabel">
       <property name="text">
        <string>Value (m)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="d2ValueLabel">
       <property name="text">
        <string>Value (m)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="d2NameEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="d1NameEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="d3ValueLabel">
       <property name="text">
        <string>Value (m)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="d3NameEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="tLabel">
       <property name="text">
        <string>Thickness (m)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QPushButton" name="displaydxfButton">
       <property name="text">
        <string>Display Model Image</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="elSizeMaxLabel">
       <property name="text">
        <string>0.5 (Max)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QRadioButton" name="elSizeRefineRadio">
       <property name="text">
        <string>Refine</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QRadioButton" name="paramVaryARadio">
       <property name="text">
        <string>Vary</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QComboBox" name="unitsComboBox">
       <item>
        <property name="text">
         <string>SI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IMPERIAL</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="reportEdit">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>260</y>
      <width>401</width>
      <height>189</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>270</width>
      <height>215</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="elasticModLabel">
         <property name="text">
          <string>Elastic Modulus (GPa)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="elasticEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="poissonRatLabel">
         <property name="text">
          <string>Poisson's Ratio</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="poissonEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="fNameLabel">
         <property name="text">
          <string>Force Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fNameEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="fEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="forceAngleLabel">
         <property name="text">
          <string>Angle (deg)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="fAngleEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fLabel">
         <property name="text">
          <string>Force (N/m)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="bcNameLabel">
         <property name="text">
          <string>Boundary Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="bcNameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="bcLabel">
         <property name="text">
          <string>Boundary Value (m)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="bcEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
    <addaction name="actionLoadDXF"/>
   </widget>
   <widget class="QMenu" name="menuCalc">
    <property name="title">
     <string>Calc</string>
    </property>
    <addaction name="actionExecute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionLoadDXF">
   <property name="text">
    <string>Load DXF File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
